fixieai/agents-ts/src/template.ts

```diff
@@ -9,26 +9,27 @@
  *     https://github.com/fixie-ai/fixie-examples
  */
 
-// @ts-expect-error
-import { CodeShotAgent, Message } from 'fixieai';
+export const BASE_PROMPT = `I'm an agent that rolls virtual dice!`;
```

Comment:
We should probably align the Python/TS templates -- @hessamb might have ideas on how to make the template easier to reason about if it's run without changes.